mkt/users/tasks.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+from django.db import transaction
+
+import mkt
+from mkt.access.models import Group, GroupUser
+from mkt.api.models import Access
+from mkt.users.models import UserProfile
+
+
```

Comment:
maybe mkt/users/utils.py? That exists already. Celery reads in the tasks.py file of each app looking for tasks so it seems odd to be here.